setup-scripts/setupProvider.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+kubectl apply -f ../providers/github/install.yaml
+kubectl apply -f ../providers/github/provider-secret.yaml
+kubectl apply -f ../providers/github/provider-config.yaml
```

Comment:
The provider takes time to install, without waiting for the resource to be ready the provider config apply will fail. 

What do you think of waiting until the provider is healthy, before installing the provider config?

`kubectl wait "provider.pkg.crossplane.io/provider-github" --for=condition=healthy --timeout=600s`